README.md

```diff
@@ -73,3 +73,32 @@ This is useful for doing things like enabling strict mode only in the theme edit
 
 It is recommended that you enable `:strict` or `:warn` mode on new apps to stop invalid templates from being created.
 It is also recommended that you use it in the template editors of existing apps to give editors better error messages.
+
+### Undefined variables and filters
+
+By default, the renderer doesn't raise or in any other way notify you if some variables or filters are missing, i.e. not passed to the `render` method.
+You can improve this situation by passing `strict_variables: true` and/or `strict_filters: true` options to `render` method.
+When one of these options is set to true, all missing errors will be stored in `errors` array of a `Liquid::Template` instance.
+Here are some examples:
+
+```ruby
+template = Liquid::Template.parse("{{x}} {{y}} {{z.a}} {{z.b}}")
+template.render({ 'x' => 1, 'z' => { 'a' => 2 } }, { strict_variables: true })
+template.errors
+#=> [#<Liquid::MissingVariable: Liquid error: missing variable y>, #<Liquid::MissingVariable: Liquid error: missing variable b in {"a"=>2}>]
```

Comment:
Example still has the `MissingVariable`. I think you renamed that. Can you update here?